api/app/controllers/application_controller.rb

```diff
@@ -6,6 +6,10 @@ class ApplicationController < ActionController::API
   protected
 
   def configure_permitted_parameters
+    # For sign-up
     devise_parameter_sanitizer.permit(:sign_up, keys: %i[name birth_date])
+
+    # For account updates
+    devise_parameter_sanitizer.permit(:account_update, keys: %i[bio avatar])
```

Comment:
```suggestion
    devise_parameter_sanitizer.permit(:sign_up, keys: %i[name])
    devise_parameter_sanitizer.permit(:account_update, keys: %i[birth_date bio avatar])
```

Sacaría los comments también porque medio que son redundantes a lo que la configuración ya explica 🙂